ultrasonic/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
           package="org.moire.ultrasonic"
           android:installLocation="auto">
 
+    <uses-sdk android:minSdkVersion="20" android:targetSdkVersion="29" />
```

Comment:
I'm not actually sure what a good version is. This change, and all the rest in this file come out of [this doc here](https://developer.android.com/training/cars/media/auto) and [this one](https://developer.android.com/training/cars/media).